docs/en/operation/StressBench.md

```diff
@@ -263,3 +263,542 @@ Here is a sample usage demo, where we have one master and three workers:
+    </tr>
+    <tr>
+        <td>warmup</td>
+        <td>15s</td>
+        <td>The length of time to warmup before recording measurements. (1m, 10m, 60s, 10000ms, etc.)</td>
+    </tr>
+    <tr>
+        <td>duration</td>
+        <td>30s</td>
+        <td>The length of time to run the benchmark. (1m, 10m, 60s, 10000ms, etc.)</td>
+    </tr>
+    <tr>
+        <td>conf</td>
+        <td>{}</td>
+        <td>Any HDFS client configuration key=value. Can repeat to provide multiple configuration values.</td>
+    </tr>
+    <tr>
+        <td>skip-prepare</td>
+        <td>false</td>
+        <td>If true, skip the prepare.</td>
+    </tr>
+    
+
+</table>
+
+### Single node testing
+#### Prerequisite
+* A running Alluxio cluster with one master, and one worker.
+
+#### Single test
+Run the test with the operation you want to test.
+```console
+$ bin/alluxio runClass alluxio.stress.cli.StressMasterBench --operation ListDir ...
+```
+
+For example, this would continuously run `ListDir` opeartion for 30s and record the throughput after 5s warmup.
+```console
+$ bin/alluxio runClass alluxio.stress.cli.StressMasterBench --operation ListDir --warmup 5s --duration 30s 
+```
+#### MaxThroughput test
+MaxThroughput test is the recommended way to test the master throughput for certain operation.
+```console
+$ bin/alluxio runClass alluxio.stress.cli.suit.MaxThroughput --operation CreateFile ...
+```
+
+For example, we are trying to get max throughput of CreateFile operation using hadoop compatible client.
+```console
+$ bin/alluxio runClass alluxio.stress.cli.suit.MaxThroughput --operation CreateFile --warmup 5s --duration 30s --client-type AlluxioHDFS
+```
+
+
+### Cluster testing
+#### Prerequisite
+- A running Alluxio cluster. Each worker node contains at least one worker, one job worker.
+- Ufs has enough space to store the test data if you are testing operation contains file creation.
+
+#### Testing
+The cluster testing is similar to single node testing except that
+- `--cluster` argument needs to be added to each operation so that the bench jobs will be submitted to job master, distributed to job workers, and executed by job workers.
+  Each job worker executes one bench job. Cluster throughput is calculated by aggregating the read throughput of each bench job.
+
+### Limitations
+- Master Stress Bench only supports testing self-generated test files.
+
+## Worker Stress Bench
```

Comment:
Doesn't get much information about worker stress bench and we are not using it currently.